src/slave/containerizer/mesos/isolators/cgroups2/cgroups2.cpp

```diff
@@ -30,6 +32,31 @@ namespace mesos {
 namespace internal {
 namespace slave {
 
+namespace cgroups2_isolator {
+
+Try<bool> supported()
+{
+  #ifndef ENABLE_CGROUPS_V2
+    return false;
+  #endif
+  if (!::cgroups2::enabled()) {
```

Comment:
This checks if the host supports the `cgroup2` filesystem type. The name `cgroups2::enabled()` could be better.